pipelines/internal/commands/run/run.go

```diff
@@ -327,13 +328,19 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 	}
 
 	var actions []*genomics.Action
+	if *outputInterval != 0 && *output != "" {
+		action := bash(fmt.Sprintf("while true; do gsutil -q cp /google/logs/output %s; sleep %.0f; done", *output, (*outputInterval).Seconds()))
```

Comment:
We should probably do this the other way around: sleep then log (there won't be anything very interesting in this very first run)

Reply:
Makes sense. Done.